deployment/README.md

```diff
@@ -20,6 +20,18 @@ the project:
 gcloud auth application-default login
 ```
 
+Then, specify your Google Cloud project ID:
```

Comment:
This GCP specific stuff should probably go into a README in the `live/example-gcp` directory (my fault - I accidentally put the `gcloud` line in above)